recipes/qhull/build.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/sh
+
+CMAKE_GENERATOR="Unix Makefiles"
+
+cmake -G "$CMAKE_GENERATOR" \
+    -DCMAKE_BUILD_TYPE:STRING=Release \
+    -DCMAKE_INSTALL_PREFIX:PATH=${PREFIX} 
+
```

Comment:
Have you tried building shared libraries by adding `-DBUILD_SHARED_LIBS=yes`? It's better to build shared libraries if it is supported.

Reply:
qhull, both shared / static libraries are generated at build.
https://github.com/qhull/qhull/blob/master/CMakeLists.txt#L333-L347